hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/AbstractMapDataStore.java

```diff
@@ -77,8 +78,8 @@ protected Object toObject(Object obj) {
         return serializationService.toObject(obj);
     }
 
-    protected Data toData(Object obj) {
-        return serializationService.toData(obj);
+    protected Data toHeapData(Object obj) {
+        return ToHeapDataConverter.toHeapData(serializationService.toData(obj));
```

Comment:
What is the advantage of `ToHeapDataConverter `?

Reply:
fixed